26-register_publication/1659-creator_navigates_to_the_reference_tab_and_selects_reference_subtype.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Creator navigates to the Reference tab and selects Reference subtype
+
+    @1625
+    Scenario: Creator navigates to the Reference tab and selects Reference subtype
+        Given Creator begins registering a Publication in the Wizard
+        When they navigate to the Reference tab
+        And they select the Reference type "Contribution to journal"
+        And they select one of Reference subtypes:
+            | Short communication  |
+            | Letter to the Editor |
+            | Book review          |
+            | Editorial            |
+        Then they see mandatory fields:
+            | Search box for Journal |
+        And they see optional fields:
+            | DOI            |
+            | Volume         |
+            | Issue          |
+            | Pages from     |
+            | Pages to       |
+            | Article number |
```

Comment:
er vi sikre på at alle disse subtypene skal ha samme felt?

Reply:
ja, eneste forskjell for alle subtypenetypene til Journal atm er at JournalArticle kan være peer reviewed, og Corrigendum skal kunne søke etter en JournalArticle